### PR TITLE
Update UG

### LINKED ISSUE #425: Update screenshots for command format

### LINKED ISSUE #438: Update Command Summary
- missing swift tag 
-examples might be wrong

### PR DESCRIPTION
**Pull request by zhonghan721**
Update formatting for Command Summary
Update screenshot for ID
Add disclaimer for ID
Update index.md to mention HomeBoss
Fix wrong diagram path in DG

#402 #404 #406 #409

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -11,10 +11,13 @@
...
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**HomeBoss is a desktop application for home businesses who need to manage their deliveries for their customers.** 
+While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
 
-* If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
-* If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
+* If you are interested in using HomeBoss,
+  head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
```
Comment (B-enguin) [inline docs/index.md]: Should this be the getting started instead

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1067,44 +1070,50 @@ A: HomeBoss only supports English characters.
...
+| View    | `customer view CUSTOMER_ID`{.swift}                                                                                                   | `customer view 1`{swift}                                                                                                 |
+| List    | `customer list`{.swift}                                                                                                               | `customer list`{.swift}                                                                                                  |
+| Find    | `customer find KEYWORD [MORE_KEYWORDS]`{.swift}                                                                                       | `customer find Julius Yang`{.swift}                                                                                      |
+| Edit    | `customer edit CUSTOMER_ID [--name NAME] [--phone PHONE_NUMBER] [--email EMAIL] [--address ADDRESS]`{.swift}                          | `customer edit 1 --name Gabriel -–phone 12345678 --email Gabrielrock@gmail.com --address Block 10 Tampines Road`{.swift} |
```
Comment (B-enguin) [inline docs/UserGuide.md]: I think the dash for the example phone prefix might be different from each other

**Review by B-enguin — COMMENTED** [review]
Some general comments

File: docs/index.md (hunk 1 above)
Comment (zhonghan721) [reply docs/index.md]: Good catch! Have changed it accordingly.

File: docs/UserGuide.md (hunk 2 above)
Comment (zhonghan721) [reply docs/UserGuide.md]: Good catch! Have changed to the correct dash

**Review by juliusgambe — APPROVED** [review]
LGTM

**Pull request merged**